changelog.md

```diff
@@ -5,6 +5,9 @@
 **Added:**
 - bpk-theming:
   - New web theming utility, see https://backpack.github.io/components/utilities/theming
+- bpk-component-theme-switcher:
+  - New component to help switching themes on the fly
+
```

Comment:
We might not need to mention this, if it's not particular user-facing.

Reply:
Shall I remove this from the changelog? Again I've copied over what has been done for RTL
cc @matthewdavidson @k0nserv

Reply:
My take is that if we want it to be public we should make it more configurable and generic